### PR TITLE
[TF 2.0 API Docs] Edited Documentation to tf.dtypes.cast

### LINKED ISSUE #27657: [TF 2.0 API Docs] tf.keras.activations.selu
**System information**
- TensorFlow version: 2.0 alpha
- Doc Link:
https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/activations/selu
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/activations.py

**- Links**
Should format the referenced paper - get rid of "-" and add the authors/year of publishing:
`"Self-Normalizing Neural Networks" (Klambauer et al, 2017)"`

**- Definition**
The current definition does not specify the fixed values for `alpha` and `scale` constants. It also assumes knowledge of the ELU activation function. We should also include a link (i.e. https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/activations/elu).

Proposed modified definition:

```
The Scaled Exponential Linear Unit (SELU) activation function is:

`scale` * `x` if `x > 0` and `scale * alpha * (exp(x)-1)` if `x < 0`

where `alpha` and `scale` are pre-defined constants (`alpha = 1.6732632423543772848170429916717` and `scale = 1.0507009873554804934193349852946`.
The SELU activation function multiplies  `scale` > 1 with the `[elu](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/activations/elu)` (Exponential Linear Unit (ELU)) to ensure a slope larger than one for positive net inputs. 
```

Followed by what is already in the docs with the formatted `lecun_normal` initialization bit and a link to it: https://www.tensorflow.org/api_docs/python/tf/keras/initializers/lecun_normal:

```
...The values of alpha and scale are chosen so that the mean and variance of the inputs are preserved between two consecutive layers as long as the weights are initialized correctly (see `[lecun_normal` initialization](https://www.tensorflow.org/api_docs/python/tf/keras/initializers/lecun_normal)) and the number of inputs is "large enough" (see references for more information).
```

**- Examples**
Can add a modified example fro the Intro to CNNs tutorials (use `selu` instead of `relu`)

```
model = models.Sequential()
model.add(layers.Conv2D(32, (3, 3), activation='selu', input_shape=(28, 28, 1)))
model.add(layers.MaxPooling2D((2, 2)))
model.add(layers.Conv2D(64, (3, 3), activation='selu'))
model.add(layers.MaxPooling2D((2, 2)))
model.add(layers.Conv2D(64, (3, 3), activation='selu'))
```

**- Returns**
Can modify to include the word `function`:
```
The scaled exponential unit activation function`: `scale * elu(x, alpha)`.
``` 
and format markdown for plain text.

**- Raises**
Not defined.

**- Visuals**
Should be added, similar to: https://cdn-images-1.medium.com/max/1600/1*WyQS-lnoemRA3_FpRL7r5w.png

**We welcome contributions by users. Will you be able to update submit a PR (use the [doc style guide](https://www.tensorflow.org/community/documentation)) to fix the doc Issue?**
Yes

### LINKED ISSUE #29210: [TF 2.0 API Docs] tf.dtypes.cast
Thank you for submitting a TensorFlow documentation issue. Per our GitHub
policy, we only address code/doc bugs, performance issues, feature requests, and
build/installation issues on GitHub.

The TensorFlow docs are open source! To get involved, read the documentation
contributor guide: https://www.tensorflow.org/community/contribute/docs

## URL(s) with the issue:

https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/dtypes/cast

### Usage example

Is there a usage example?
the example: 
```python
x = tf.constant([1.8, 2.2], dtype=tf.float32)
tf.cast(x, tf.int32)  # [1, 2], dtype=tf.int32
```
is not correct.  
It need to change ```tf.cast``` to ```tf.dtypes.cast```.

this is correct example
```python
x = tf.constant([1.8, 2.2], dtype=tf.float32)
tf.dtypes.cast(x, tf.int32)  # [1, 2], dtype=tf.int32
```

### Submit a pull request?

https://github.com/tensorflow/tensorflow/pull/29214

Are you planning to also submit a pull request to fix the issue? See the docs
contributor guide: https://www.tensorflow.org/community/contribute/docs and the
docs style guide: https://www.tensorflow.org/community/contribute/docs_style

### PR DESCRIPTION
**Pull request by kooock**
Fix #29210 by changing example to tf.dtypes.cast.

### CODE REVIEW

**Comment by rthadur** [comment]
@kooock can you please sign CLA.

**Review by dynamicwebpaige — APPROVED** [review]
Thanks for the update! 😊

**Comment by kooock** [comment]
I signed it!
